### PR TITLE
Added brave path to Makefile

### PR DESCRIPTION
**Pull request by BachoSeven**

### CODE REVIEW

**Comment by alexherbo2** [comment]
Re. o/

The last time I tested the application on [Brave], it worked with the [Chromium path][Native messaging host location].

Do you have a link where this path is documented?

[Brave]: https://brave.com
[Native messaging host location]: https://developer.chrome.com/extensions/nativeMessaging#native-messaging-host-location

**Comment by BachoSeven** [comment]
I did not find any mentions of the Linux path on the web; but the reason
I did this is that Brave (at least the nightly build which I use) now
has a folder inside ${XDG_CONFIG_HOME:-HOME/.config}/BraveSoftware/Brave-Browser/NativeMessagingHosts
by default, so I thought it was natural to put it there.

I just tested with Chromium and Chrome locations and the shell extension won't
load, error explicitly mentions NativeMessagingHosts' path.

File: host/Makefile [hunk 1]
```diff
@@ -27,7 +30,7 @@ install: $(build)
 	./scripts/install-manifest chrome chromium firefox
 
 uninstall:
-	./scripts/uninstall-manifest chrome chromium firefox
+	./scripts/uninstall-manifest brave chrome chromium firefox
```
Comment (alexherbo2) [inline host/Makefile:33]: After Chromium.

File: host/Makefile [hunk 2]
```diff
@@ -27,7 +30,7 @@ install: $(build)
 	./scripts/install-manifest chrome chromium firefox
```
Comment (alexherbo2) [inline host/Makefile:30]: Add Brave.

File: host/Makefile [hunk 3]
```diff
@@ -8,6 +8,9 @@ endif
 
 all: chrome chromium firefox
 
+brave: $(build)
```
Comment (alexherbo2) [inline host/Makefile:11]: After Chromium.

File: host/scripts/brave-target.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Environment variables
+XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config}
+
+PLATFORM='chrome'
+
+# Native messaging host location
+# https://developer.chrome.com/extensions/nativeMessaging#native-messaging-host-location
+    TARGET="$XDG_CONFIG_HOME/BraveSoftware/Brave-Browser/NativeMessagingHosts"
```
Comment (alexherbo2) [inline host/scripts/brave-target.sh:8]: Add support for Linux and macOS.

File: host/scripts/brave-target.sh [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Environment variables
+XDG_CONFIG_HOME=${XDG_CONFIG_HOME:-~/.config}
+
+PLATFORM='chrome'
+
+# Native messaging host location
+# https://developer.chrome.com/extensions/nativeMessaging#native-messaging-host-location
```
Comment (alexherbo2) [inline host/scripts/brave-target.sh:7]: Update the link.

File: host/Makefile [hunk 6]
```diff
@@ -8,6 +8,9 @@ endif
 
 all: chrome chromium firefox
```
Comment (alexherbo2) [inline host/Makefile:9]: Add Brave.

**Pull request closed without merging**